data/autoyast_sle15/autoyast_containers_aarch64.xml

```diff
@@ -102,6 +102,13 @@
     <install_recommended config:type="boolean">true</install_recommended>
     <packages config:type="list">
       <package>iputils</package>
+      <package>sles-release</package>
```

Comment:
do we really need to explicitly install this one ?

Reply:
it was in other xmls, and I think it was copy-pasted from another autoyast profile. Not sure if it's really needed and what for..